src/components/Ellipsed.tsx

```diff
@@ -57,10 +57,10 @@ const Ellipsed = ({ text, className }: EllipsedProps) => {
 
   const handleResize = createHandleResize(el, charWidth, text, setText)
 
-  useLayoutEffect(() => {
+  useEffect(() => {
```

Comment:
Why this change?

Reply:
To solve the issue where the hash of the selected address was not changing in the Send modal (AddressSelect) (demonstrated in the video). I had to put `text` into the depedencies of a `useEffect` and since the body of it was the same as the one in `useLayoutEffect`, I merged them into one `useEffect`. I read the docs for `useLayoutEffect` and it says to always prefer `useEffect` unless you really need to use `useLayoutEffect`. Since everything still works, I went for the simpler solution. Did I miss sth?

Reply:
`useLayoutEffect` was purposefully used because the measurements needed for the ellipsis must occur after DOM updates - it may appear to work in this instance but the problem could manifest at another time - this happened to me while I was working on it

The dependency is ok :ok_hand: